ecommerce/management/commands/invalidate_payment_coupons.py

```diff
@@ -0,0 +1,79 @@
+"""
+Disables coupons in a batch, either by parsing a CSV file or by iterating
+through a coupon payment. 
```

Comment:
This is fine too, But can we mention the command option here as well just for more clarity, I think we can copy past the same things in here that you have mentioned in the PR description.